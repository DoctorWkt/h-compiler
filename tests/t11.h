int c[]= { 1,2,3,4 };

void main()
{
  int i;
  for (i=0; i!= 4; i++)
    printn( c[i] );
  putchar('\n');

  for (i=0; i!= 4; i++) 
    c[i]++;

  for (i=0; i!= 4; i++)
    printn( c[i] );
  putchar('\n');

  for (i=0; i!= 4; i++) 
    c[i]--;

  for (i=0; i!= 4; i++)
    printn( c[i] );
}
