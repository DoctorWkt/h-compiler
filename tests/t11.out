000001
000002
000003
000004

000002
000003
000004
000005

000001
000002
000003
000004
